src/scene/materials/standard-node-material.js

```diff
@@ -0,0 +1,100 @@
+import { standard } from '../../graphics/program-lib/programs/standard.js';
+
+import {
+    SHADER_FORWARDHDR, SHADER_PICK
+} from '../constants.js';
+
+import { StandardMaterial } from './standard-material.js';
+
+/**
+ * @private
+ * @class
+ * @name pc.StandardNodeMaterial
+ * @augments pc.StandardMaterial
+ * @classdesc StandardNodeMaterial is sub class of the StandardMaterial, and adds shader graph interop functionality
+ * @param {pc.StandardMaterial} mat - optional material which is cloned
+ * @param {any} chunk - optional shader graph node chunk to be used
+ */
+function StandardNodeMaterial(mat, chunk) {
+    StandardMaterial.call(this);
+
+    if (mat) {
+        StandardMaterial.prototype._cloneInternal.call(mat, this);
+    }
+
+    if (chunk) {
+        this._shaderGraphChunk = chunk;
+    }
```

Comment:
remove the if line, so _shaderGraphChunk is always assigned to (either valid chunk, or undefined)

Reply:
true - will remove